package.json

```diff
@@ -228,7 +228,7 @@
     "taskr": "1.1.0",
     "tree-kill": "1.2.2",
     "tsec": "0.2.1",
-    "turbo": "1.10.9",
+    "turbo": "1.11.3",
```

Comment:
We need to update our CI files as well

x-ref: https://github.com/vercel/next.js/blob/21452921c055421ec8a44dae6e0e5f311c503337/.github/workflows/build_reusable.yml#L51
https://github.com/vercel/next.js/blob/21452921c055421ec8a44dae6e0e5f311c503337/.github/workflows/build_and_test.yml#L11
https://github.com/vercel/next.js/blob/21452921c055421ec8a44dae6e0e5f311c503337/.github/workflows/build_and_deploy.yml#L10